src/wallet/rpc/backup.cpp

```diff
@@ -1589,7 +1589,8 @@ RPCHelpMan importdescriptors()
     return RPCHelpMan{"importdescriptors",
                 "\nImport descriptors. This will trigger a rescan of the blockchain based on the earliest timestamp of all descriptors being imported. Requires a new wallet backup.\n"
             "\nNote: This call can take over an hour to complete if using an early timestamp; during that time, other rpc calls\n"
-            "may report that the imported keys, addresses or scripts exist but related transactions are still missing.\n",
+            "may report that the imported keys, addresses or scripts exist but related transactions are still missing.\n"
+            "The rescan is significantly faster if block filters are available (bitcoind option \"-blockfilterindex=1\").\n",
```

Comment:
In 1f0198a3fe58d42dce9e8ddaa09dc2ba8171930e "rpc: doc: mention rescan speedup using `blockfilterindex=1` in affected wallet RPCs"

I think we generally try to avoid using our binary names directly in help text like this.

```suggestion
            "The rescan is significantly faster if block filters are available (Using startup option \"-blockfilterindex=1\").\n",
```